ideas/290-hardware-wallet-light

```diff
@@ -0,0 +1,123 @@
+---
+id: 290-hardware-wallet-light
+title: Hardware Wallet Light
+status: Active
+created: 2018-08-24
+category: core
+lead-contributor: bitgamma
+contributors:
+    bitgamma
+    gravityblast
+    goranjovic
+    dmitryn
+    patrick
+    denis-sharpyn
+    guylouis
+    
+exit-criteria: true
+success-metrics: true
+clear-roles: true
+future-iteration: true
+roles-needed:
+    
+---
+
+## Preamble
+
+    Idea: 290-hardware-wallet-light
+    Title: Hardware Wallet Light
+    Status: In Progress
+    Created: 2018-08-24
+
+## Summary
+
+We have a Javacard based wallet implementation ready, see https://github.com/status-im/hardware-wallet
+
+This swarm is about making the hardwallet available (uxr, ux, client integration, sourcing) to our Android users.
+
+## Swarm Participants
+
+- Lead Contributor: @bitgamma
+- Go: @gravityblast
+- Clojure: @gorandjovic @dmitryn
+- UXR: @patrick
+- UX: @denis-sharpyn
+- PM:@guylouis
+
+## Product Overview
+
+The product is a HD hardware wallet in the form of a credit card size javacard with contactless capability (nfc), and is natively usable with our android clent.
+
+It will provide extended security and convenience to our users, providing:
+- ability to create or import an existing Status account or any BIP-32 wallet onto the javacard
```

Comment:
for the first step can we only create new accounts? I think we are still not sure about importing existing accounts unless we ask the mnemonic.